doc/calculator/Cargo.toml

```diff
@@ -12,7 +12,7 @@ path = "../../lalrpop"
 features = ["lexer"]
 
 [dependencies]
-regex = "1"
```

Comment:
This is a source code included as example in the doc. To let users copy it easily, isn't it expected to be independent with the workspace?

Reply:
Ah, whoops